deposit_rental/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Deposit Rental",
+    "description": "Adds deposit functionality for rental products.",
+    "category": "Rental/Deposit",
+    "depends": ["sale_renting","website_sale"],
```

Comment:
```suggestion
    "depends": ["sale_renting", "website_sale"],
```

Why not `website_sale_renting` instead?